SimulationRuntime/c/simulation_data.h

```diff
@@ -47,7 +47,7 @@
 #define omc_dummyVarInfo {-1,-1,"","",omc_dummyFileInfo}
 #define omc_dummyEquationInfo {-1,0,"",-1,NULL}
 #define omc_dummyFunctionInfo {-1,"",omc_dummyFileInfo}
-#define omc_dummyRealAttribute {NULL,NULL,DBL_MAX,-DBL_MAX,0,0,1.0,0,0.0}
+#define omc_dummyRealAttribute {NULL,NULL,DBL_MAX,-DBL_MAX,0,0,1.0,0.0}
```

Comment:
Are you sure that `modelica_real min; /* = -Inf */` is `DBL_MAX` and
`modelica_real max; /* = +Inf */` is `-DBL_MAX`?